addons/l10n_my_edi/tests/test_file_generation.py

```diff
@@ -339,6 +339,52 @@ def test_06_foreigner(self):
             self.partner_b.commercial_partner_id.l10n_my_identification_number,
         )
 
+    def test_07_tax_exempt(self):
+        """
+        Check that the file is correct with an exempt tax.
+        """
+        exempt_tax = self.env['account.tax'].create({
+            'name': 'Tax Exempt',
+            'type_tax_use': 'sale',
+            'amount_type': 'percent',
+            'amount': 0,
+            'l10n_my_tax_type': 'E',
+        })
+        invoice = self.init_invoice(
+            'out_invoice', partner=self.partner_b, products=self.product_a, taxes=exempt_tax,
+        )
+        invoice.l10n_my_edi_exemption_reason = "Exempt Customer"
+        invoice.action_post()
+
+        file, errors = invoice._l10n_my_edi_generate_invoice_xml()
+        self.assertFalse(errors)
```

Comment:
```suggestion
        self.assertEqual(errors, set())
```
Makes more sense given `_l10n_my_edi_generate_invoice_xml()`'s return types, and is more consistent with other similar tests :nerd_face:

Reply:
It was on purpose; not long ago, WAN gave me a comment on that kind of assert. (https://github.com/odoo/odoo/pull/193497#discussion_r1928519521)

Here, I only care that no errors are returned, so IMO it's fine like this (I agree that it's not consistent with the rest of the file, but I can maybe align everything in master in the PR that merges both my_edi modules)

I can, of course, change it if you prefer.

Reply:
No, it's ok, don't worry, it was merely a suggestion.
But if you can align the syntax in `master`, it would be great :ok_hand: